documentation/sphinx/source/data-modeling.rst

```diff
@@ -269,6 +269,22 @@ Using the table name as the subspace, we could implement the common row-oriented
             cols[c] = v
         return cols
 
+
+Versionstamps
+-------------
+
+A common data model is to index your data with a sequencing prefix to allow log scans or tails of recent data. This index requires a unique, monotonically increasing value. This could be implemented at the client level by reading the value for conflict checks before every increment. A better solution is the versionstamp, which can be generated at commit-time with no read conflict ranges, providing a unique sequence ID in a single conflict-free write.
```

Comment:
I think it'd be worth making your allusion more explicit that versionstamps are the replacement for AUTOINCREMENT PRIMARY KEY.

Reply:
Got it.  versionstamp is FDB's version of to an AUTOINCREMENT PRIMARY KEY in SQL

Reply:
Is there a change that is associated with this comment?

Reply:
Whoops, there was in my head!  Just pushed